src/SFA.DAS.AdminService.Web/Controllers/HomeController.cs

```diff
@@ -29,7 +34,7 @@ public IActionResult Error()
 
         public IActionResult InvalidRole()
         {
-            return View();
+            return View(new Error403ViewModel(_configuration["ResourceEnvironmentName"]) { UseDfESignIn = _webConfiguration.UseDfESignIn });
```

Comment:
I have never seen this done before using resource environment name from the azure configuration, you should be using configuration values from the storage configuration

Reply:
As the ResourceEnvironemtName is a deployment setting and it change based on the current deployment environment, Values are configured in the Pipeline Library, Similar to the configuration["EnvironmentName"]

Reply:
Changed the help link from configuration. So this change is not required anymore.